.github/workflows/frontend-build-and-publish-apps.yml

```
@@ -160,111 +166,39 @@ jobs:
           echo "name=${BRANCH_NAME}" >> "$GITHUB_OUTPUT"
           echo "::notice ::Branch name: $BRANCH_NAME"
 
+      - name: Package Json Metadata
+        id: package-json
+        run: |
+          echo "build-solution=$(node -p "require('${{ inputs.project-path }}package.json').vsBuildSolution")" >> "$GITHUB_OUTPUT"
+          echo "name=$(node -p "require('${{ inputs.project-path }}package.json').name")" >> "$GITHUB_OUTPUT"
+          echo "version=$(node -p "require('${{ inputs.project-path }}package.json').version")" >> "$GITHUB_OUTPUT"
+
+      - name: Generate Version Number
+        id: version-builder
+        uses: River-iGaming/actions/dotnet/versions@feature/fe-support
```

```suggestion
        uses: River-iGaming/actions/dotnet/versions@v2
```
once PR is merged https://github.com/River-iGaming/actions/pull/10/files